introduction.md

```diff
@@ -1,16 +1,2 @@
 Hi, I am Daniel!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?